doc/api_assets/api.js

```diff
@@ -2,24 +2,25 @@
 
 {
   function setupTheme() {
-    const kCustomPreference = 'customDarkTheme';
-    const userSettings = sessionStorage.getItem(kCustomPreference);
+    const storedTheme = localStorage.getItem('theme');
     const themeToggleButton = document.getElementById('theme-toggle-btn');
 
-    if (userSettings === null && window.matchMedia) {
+    // Follow operating system theme preference
+    if (storedTheme === null && window.matchMedia) {
       const mq = window.matchMedia('(prefers-color-scheme: dark)');
 
       if ('onchange' in mq) {
+
         function mqChangeListener(e) {
           document.documentElement.classList.toggle('dark-mode', e.matches);
         }
+
         mq.addEventListener('change', mqChangeListener);
+
         if (themeToggleButton) {
           themeToggleButton.addEventListener(
             'click',
-            function() {
-              mq.removeEventListener('change', mqChangeListener);
```

Comment:
okay

Reply:
Done. I think it is worth having some linter rules related to empty lines to automate such stuff and reduce waste...

Reply:
This change is still there.

Reply:
I've removed the empty lines, but this one was kept to improve readability. I believe having oneliner here instead of 3 lines is useful here imo.
Don't you think so, @anonrig?

Reply:
> Done. I think it is worth having some linter rules related to empty lines to automate such stuff and reduce waste...

Probably, but unrelated. As much as I really appreciate empty lines, on node core we avoid as much as possible unrelated changes, including cosmetic ones.

Reply:
> I've removed the empty lines, but this one was kept to improve readability. I believe having oneliner instead of 3 lines is useful here imo.
> 
> Don't you think so, @anonrig?

This is unrelated to your changes. Please either commit them differently or open a new pull request with this stylistic changes.

Reply:
Ok, thanks. Reverted.

Reply:
IMO it's just a nit...